RoaringBitmap/src/main/java/org/roaringbitmap/buffer/ImmutableRoaringBitmap.java

```diff
@@ -1458,7 +1457,6 @@ public boolean isEmpty() {
    *
    * @return the iterator
    */
-  @Override
   public Iterator<Integer> iterator() {
```

Comment:
I feel the method should be dropped alltogether, or at least marked `@Deprecated`